exercise-1.js

```diff
@@ -1,199 +1,296 @@
-            return [];
-        }
-
-
-
-
-
-
-
-        /************************************************************************************/
-        // Write a function that reverses your telephone number.  It should return the reversed telephone number.
-
-        //your code...
-        function reversePhone(number) {
-            
-        }
-
-
-
-
-
-        /************************************************************************************/
-        // Write a function that returns a car object using some given info about your car. Required inputs are the make, model, year, and color.
-
-        //your code...
-
-
-
-
-
-
-        /************************************************************************************/
-        // Write a function that accepts a list of numbers.  The function should identify each number as being even or odd.  The function should output a set of key value pairs, with the key being the number, and the value being the string "even" or "odd".
-
-        //example : [10,23,3,4] => function() => {10 : 'even', 23 : 'odd', 3 : 'odd', 4 : 'even'}
-
-        //your code...
-
-
-
-
-
-
-
-        /************************************************************************************/
-        const numbers = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-        // Write a "for" loop that console.log()'s the first value in the  array, and every 3rd number, i.e. 0, 3, 6, 9: the zeroth, third, sixth, and ninth values.
-
-        //your code...
-
-
-
-        /************************************************************************************/
-        const foodArray = [ 'potatoes', 'tamales', 'lemon','strawberries','chocolate', 'pudding', {program : 'TEKcamp'} ];
-        //access the value of the last element of the array and set it to a variable called school.  print the school variable to the console.
-
-        const adjectiveArray = [ 'salty', 'spicy', 'sour', 'sweet', 'rich','creamy','amazing'];
-        // Using both the foodArray and the adjectiveArray, write "for" loop that console.log()'s a sentence for each corresponding value in the arrays. Add the word "is" or "are" depending on if the food is singular or plural.  i.e. "Potatoes are salty", "Lemon is sour".
-
-
-        /************************************************************* */
-        // Refactor the for() loop to be a while loop.
-
-
-        for(let i=0; i<10; i++) {
-            // console.log(" the value of i in the loop is : " + i);
-        }
-
+/************************************************************************************/
+// Write a JavaScript function that iterates the integers from 1 to 100. For multiples of 3 print "TEK" instead of the number and for multiples of 5, print "camp." For numbers that are multiples of both 3 and 5, print "TEKcamp."
+
+//your code...
+// 
+for (let i = 1; i <= 100; i++) {
+  if (i % 3 === 0) {
+    console.log("TEK");
+  } if (i % 5 === 0) {
+    console.log("camp");
+  } if (i % 3 === 0 && i % 5 === 0) {
```

Comment:
We really don't want to have more than one "if" statement at a time. Generally, we would want to use "if else" and "else" after the first if. Multiple "if" statements can cause bugs.